### PR TITLE
Bot-formatting-card issue fixes

### PR DESCRIPTION
**Pull request by Pawan-MSFT**

### CODE REVIEW

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments

File: samples/bot-formatting-cards/csharp/BotFormattingCards/Bots/DialogBot.cs [hunk 1]
```diff
@@ -66,9 +72,44 @@ public override async Task OnTurnAsync(ITurnContext turnContext, CancellationTok
...
+                    {
+                        await turnContext.SendActivityAsync("Member not found.");
+                        return;
+                    }
+                    else
+                    {
+                        throw e;
+                    }
+                }
+                var attachments = new List<Attachment>();
```
[the remark this answers is not in the document]
Comment (Pawan-MSFT) [reply samples/bot-formatting-cards/csharp/BotFormattingCards/Bots/DialogBot.cs:95]: Fixed

File: samples/bot-formatting-cards/csharp/BotFormattingCards/Cards/AllCards.cs [hunk 2]
```diff
@@ -57,7 +65,7 @@ public static Attachment sendInfoMasking()
         public static Attachment sendFullWidthCardAdaptiveCard()
         {
             var paths = new[] { ".", "Resources", "sampleAdaptiveWithFullWidth.json" };
-            var adaptiveCardJson = File.ReadAllText(Path.Combine(paths));
+            var adaptiveCardJson = System.IO.File.ReadAllText(Path.Combine(paths));
```
[the remark this answers is not in the document]
Comment (Pawan-MSFT) [reply samples/bot-formatting-cards/csharp/BotFormattingCards/Cards/AllCards.cs:68]: Fixed

**Review by ChetanSharma-msft — APPROVED** [review]
Looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Comment by Pawan-MSFT** [comment]
Executed build pipeline
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/4818666831

**Pull request merged**